sig-service-catalog/charter.md

```diff
@@ -0,0 +1,165 @@
+  - Runs the CI builds.
+  - Maintainers have access.
+- [Jenkins](https://service-catalog-jenkins.appspot.com/)
+  - Runs end-to-end tests on a live cluster.
+  - Server managed by Michael Kibbe (Google).
+
+## Roles
+
+- Maintainers
+  - Maintainer is equivalent to the standard [Kubernetes definition of
+    Approver](https://github.com/kubernetes/community/blob/master/community-membership.md#approver).
+  - Responsible for reviewing pull requests, and approving pull requests for merge.
+  - Responsible for technical planning and stewardship of the project.
+  - New maintainers may be nominated by another maintainer, and accepted via lazy
+    two-thirds resolution amongst the chairs.
+  - Maintainers may be nominated for removal from their position by a chair,
+    and accepted via lazy two-thirds resolution amongst the chairs.
+  - Maintainers may propose changes to this charter at any time, by submitting a
+    pull request and then notifying the SIG mailing list, to be accepted via
+    lazy two-thirds resolution amongst the chairs.
+
+- Chairs
+  - Chairs are expected to perform the role of maintainer, in addition to their chair responsibilities.
+  - Chairs are listed in our [SIG
+    definition](https://github.com/kubernetes/community/tree/master/sig-service-catalog#chairs).
+  - Responsible for project administration activities within the SIG and are
+    non-technical in nature, such as organizing the weekly meetings.
+  - A chair does not have more rights, or votes, than a maintainer.
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes
+    leadership teams.
+  - All decisions amongst chairs are made using lazy consensus with a fallback
+    to a 2/3 majority vote (lazy two-thirds resolution).
+    This process is used for all decisions, such as changing chairs/maintainers
+    or modifying this charter.
+  - Chairs may nominate a new chair at any time, to be accepted via lazy
+    two-thirds resolution amongst the chairs.
+  - Chairs may decide to step down at any time.
+  - Chairs must remain active in the role and may be removed from the position
+    via lazy two-thirds resolution amongst the chairs, if they are unresponsive
+    for > 3 months or are not proactively working with other chairs to fulfill
+    responsibilities.
+  - There is no set number of Chairs.
+
+- Emeritus Chairs ([Inspired by the Helm
+  Project](http://technosophos.com/2018/01/11/introducing-helm-emeritus-core-maintainers.html))
+  - A chair who steps down may choose to take an Emeritus Chair title. This
+    confers honor on the recipient and allows them to remain associated with the
+    SIG in acknowledgement of their significant contributions.
+  - Those who attain this title are no longer expected to attend the weekly
+    meetings, share in the issue queue triage rotation, vote on policy or
+    architectural issues, or review pull requests.
+  - They are listed in our documentation as Emeritus Chairs, and we will
```

Comment:
Link to the doc?

Reply:
Fixed

https://svc-cat.io/community/#leadership